proxy/ParentSelection.cc

```diff
@@ -370,7 +370,7 @@ ParentRecord::PreProcessParents(const char *val, const int line_num, char *buf,
         } else {
           Debug("parent_select", "token: %s, matches this machine.  Marking down self from parent list at line %d", fqdn, line_num);
           hs.createHostStat(fqdn);
-          hs.setHostStatus(fqdn, HostStatus_t::HOST_STATUS_DOWN, 0);
+          hs.setHostStatus(fqdn, HostStatus_t::HOST_STATUS_DOWN, 0, "manual");
```

Comment:
Can we replace these "manual"s with `Reasons::MANUAL`?

Reply:
yes, i'll make that change.